cl/recap/models.py

```diff
@@ -211,6 +212,53 @@ def file_contents(self):
     def print_file_contents(self):
         print(self.file_contents)
 
+class EmailProcessingQueue(AbstractDateTimeModel):
+    uploader = models.ForeignKey(
+        User,
+        help_text="The user that uploaded the item to RECAP.",
+        related_name="recap_email_processing_queue",
+        on_delete=models.CASCADE,
+        null=True,
+    )
+    court = models.ForeignKey(
+        Court,
+        help_text="The court where the upload was from",
+        related_name="recap_email_processing_queue",
+        on_delete=models.CASCADE,
+        null=True,
+    )
+    filepath = models.FileField(
+        help_text="The filepath to the disclosure normalized to a PDF.",
+        upload_to=make_recap_email_processing_queue_aws_path,
+        storage=AWSMediaStorage(),
+        max_length=300,
+        null=True,
+    )
+    status = models.SmallIntegerField(
+        help_text="The current status of this upload. Possible values "
+                  "are: %s"
+                  % ", ".join(
+            ["(%s): %s" % (t[0], t[1]) for t in PROCESSING_STATUS.NAMES]
+        ),
+        default=PROCESSING_STATUS.ENQUEUED,
+        choices=PROCESSING_STATUS.NAMES,
+        db_index=True,
+    )
+    error_message = models.TextField(
```

Comment:
This should be `status_message` or `error_message`. A bug in the other API is that when it works successfully the "It's all good" message is in a field called `error_message`.

Reply:
Got it, I went with consistency, and clearly you are trying to refactor 😎